task.js

```diff
@@ -91,7 +101,7 @@ class Task {
     let i = 0;
     for (const batch of batches) {
       i++;
-      const [publicGraph, privateGraph] = this.mapper(batch, translations);
+      const [publicGraph, privateGraph] = this.mapper(batch, translations,er => Task.logError(this.title,er));
```

Comment:
No need to make the function static.  If not static, you can just pass the function ` this.logError`   as argument to `this.mapper(...)`. Since `logError` is 
 a method on a task instance, you will have access  to `this.title` inside that function. There's no need to pass it as an argument.